### ISSUE TITLE
Android Basics issue:

### ISSUE THREAD

**Issue by m-elizaveta** [google-developer-training/android-basics-kotlin-tip-calculator-app-solution#273]
**URL of codelab**
https://developer.android.com/codelabs/android-basics-kotlin-write-instrumentation-tests?authuser=1&continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-kotlin-unit-2-pathway-2%3Fauthuser%3D1%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fandroid-basics-kotlin-write-instrumentation-tests#3

**In which task and step of the codelab can this issue be found?**
Step 3

**Describe the problem**
Here: `@get:Ruleval activity = ActivityScenarioRule(MainActivity::class.java)` it should perhaps say:
`@get:Rule val activity = ActivityScenarioRule(MainActivity::class.java)`
In other words I believe there is a space missing between `Rule` and `val`

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._
<img width="919" alt="Screenshot 2022-01-04 at 15 00 59" src="https://user-images.githubusercontent.com/70581050/148070263-6bfbb480-6723-410f-8a6d-e8932a0901d6.png">
